tests/common/devices/sonic.py

```diff
@@ -0,0 +1,1306 @@
+
```

Comment:
These are nice changes. However, when we break file by deleting it, and creating new files, we lose the git tracking/blame history.
Since this file (`sonic.py`) is the largest file with old code, can we not delete `tests/common/devices.py`, and instead rename it to a new file? That way we will maintain the git history for at least contents of this file. Basically:

`git mv tests/common/devices.py tests/common/devices/sonic.py`

Reply:
This is a great idea! I'll try it to keep the git history.